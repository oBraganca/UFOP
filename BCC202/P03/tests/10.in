100
I Sophia
391677 polo 10 1999 Bernardo 205
595243 gol 5 2002 Bernardo 205
735034 nivus 6 2010 Pedro 206
794242 mobi 1 1990 Gabriel 209
196192 fox 1 1997 Alice 200
663775 palio 4 2015 Helena 204
628318 corolla 3 1989 Helena 204
677740 creta 8 1992 Davi 207
498787 fox 3 2005 Davi 207
586976 polo 10 2015 Alice 200
431013 gol 8 2000 Bernardo 205
770382 hb20 2 1990 Gabriel 209
335483 hb20 8 1992 Davi 207
452051 hb20 6 2014 Helena 204
552445 nivus 3 2022 Helena 204
547314 mobi 7 2004 Helena 204
616935 onix 9 1980 Bernardo 205
661685 nivus 8 1986 Sophia 202
594839 fox 7 1987 Arthur 203
727899 fox 10 1990 Bernardo 205
665624 fox 6 1990 Pedro 206
419928 ferrari 4 2022 Arthur 203
355517 tcross 4 2020 Helena 204
509075 polo 8 1980 Pedro 206
211432 pulse 6 2021 Sophia 202
583035 palio 4 1984 Alice 200
504314 creta 6 2000 Davi 207
325031 pulse 3 1998 Arthur 203
909364 mobi 9 1998 Helena 204
940534 nivus 2 2015 Arthur 203
931010 tcross 6 1989 Pedro 206
456842 fox 7 2000 Helena 204
369965 gol 7 1988 Isabella 208
161083 palio 10 1981 Gabriel 209
367317 fox 7 2016 Arthur 203
352923 ferrari 6 1994 Isabella 208
260771 gol 1 2004 Bernardo 205
818080 polo 8 1997 Davi 207
379550 nivus 5 2022 Isabella 208
452638 mobi 1 2006 Gabriel 209
746281 yars 5 2002 Sophia 202
467372 argo 3 2017 Miguel 201
651910 ferrari 6 2001 Pedro 206
703981 corolla 2 2004 Alice 200
741104 yars 9 2019 Bernardo 205
712658 polo 10 2002 Gabriel 209
211897 onix 7 1994 Helena 204
192465 nivus 7 1999 Gabriel 209
456179 creta 7 2001 Arthur 203
981897 argo 5 2022 Alice 200
303204 argo 4 2012 Arthur 203
393692 tcross 5 2012 Gabriel 209
460789 fox 2 1999 Gabriel 209
516133 gol 2 2005 Alice 200
735283 yars 5 1986 Gabriel 209
505597 creta 1 1981 Helena 204
332529 hb20 5 2004 Davi 207
991814 ferrari 9 2006 Bernardo 205
690756 creta 3 2008 Miguel 201
959733 polo 1 2021 Isabella 208
595248 pulse 5 2007 Pedro 206
255560 yars 9 2015 Pedro 206
430493 onix 2 2022 Gabriel 209
452255 onix 8 2005 Sophia 202
683434 corolla 10 2021 Isabella 208
146133 creta 7 1993 Miguel 201
454162 hb20 3 2019 Arthur 203
545680 nivus 7 2006 Isabella 208
734909 creta 10 2006 Miguel 201
159612 mobi 6 2006 Isabella 208
811066 tcross 4 2005 Isabella 208
973326 onix 5 1993 Bernardo 205
283660 fox 8 2019 Gabriel 209
853669 fox 9 1983 Gabriel 209
199510 gol 7 2003 Alice 200
479355 tcross 10 1999 Isabella 208
194806 onix 4 1985 Pedro 206
659667 creta 6 2003 Helena 204
340105 mobi 5 1983 Davi 207
696592 argo 3 2008 Pedro 206
743804 onix 3 2015 Sophia 202
131612 polo 2 2002 Miguel 201
622540 fox 8 2015 Bernardo 205
829231 fox 1 2023 Alice 200
749649 corolla 6 1982 Helena 204
563335 hb20 2 1993 Sophia 202
765114 hb20 9 2011 Miguel 201
676573 argo 8 2011 Isabella 208
313491 ferrari 2 2000 Pedro 206
846986 creta 3 2007 Gabriel 209
687617 mobi 1 1993 Bernardo 205
919038 yars 3 2003 Helena 204
546632 corolla 7 1991 Miguel 201
882805 gol 9 1983 Isabella 208
461732 pulse 3 1990 Isabella 208
231717 ferrari 3 1999 Arthur 203
523267 pulse 7 1999 Isabella 208
947488 gol 5 1998 Bernardo 205
512714 mobi 1 2006 Davi 207
153675 argo 2 1991 Sophia 202
