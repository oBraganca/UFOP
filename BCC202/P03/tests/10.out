argo [1991|2|153675]
hb20 [1993|2|563335]
onix [2015|3|743804]
onix [2005|8|452255]
yars [2002|5|746281]
pulse [2021|6|211432]
nivus [1986|8|661685]
